autogpt_platform/backend/backend/copilot/tools/list_credentials_test.py

```
@@ -0,0 +1,456 @@
+
+    @pytest.mark.asyncio
+    async def test_registers_managed_providers_before_lookup(self, tool, mock_session):
+        registered = False
+
+        def register():
+            nonlocal registered
+            registered = True
+
+        def get_provider(_name: str):
+            assert registered
+            return None
```

Ruff (0.16.1)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)